pandas/core/groupby/ops.py

```diff
@@ -0,0 +1,1528 @@
+"""
+perform the groupby operations. These are not exposed
```

Comment:
Can you expand a bit on "perform the groupby operations". That's not directly clear to me from reading that. (it is the "split" (finding the groups) part of "split-apply-combine ?)